docs/components/dartpad/menus/theme/main.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Dropdown Menu Themed Demo',
+      debugShowCheckedModeBanner: false,
+      home: DropdownMenuDemo(),
+      theme: _buildShrineTheme(),
+    );
+  }
+}
+
+class DropdownMenuDemo extends StatefulWidget {
+  @override
+  _DropdownMenuDemoState createState() => _DropdownMenuDemoState();
+}
+
+class _DropdownMenuDemoState extends State<DropdownMenuDemo> {
+  Map<int, bool> countToValue = <int, bool>{};
```

Comment:
Same here, remove this property and make it a statelesswidget

Reply:
Done